lib/deref.js

```diff
@@ -206,6 +206,9 @@ module.exports = {
         let refResolvedSchema = this.resolveRefs(resolvedSchema, parameterSourceOption,
           components, schemaResolutionCache, resolveFor, resolveTo, stack, _.cloneDeep(seenRef), stackLimit);
 
+        if (schema.nullable) {
```

Comment:
@SahilChoudhary22 I think this only fixes issue if `schema` is having from `$ref`. For schema not having any `$ref` this does not fix the issue. (As this entire handling is only under `if (schema.$ref)` block)

i.e. try not using `$ref` in the test case below and use direct value for `$ref` and see if we get same result.

Reply:
The schema not having `$ref` and nullable is already handled in the existing code and the issue was popping up only when the parent had both nullable field and `$ref`s to another schema. 

However, as you suggested, it's a good idea to have this in our test cases, so I've added another test case in the latest commit([d9a14d8](https://github.com/postmanlabs/openapi-to-postman/pull/520/commits/d9a14d807f7702f21dda457647ee12668ec2306d)) which tests the schema that doesn't contain `$ref` and has nullable key.

Reply:
@SahilChoudhary22 Here, we are only assigning `nullable` property to `resolvedSchema`. Meanwhile, there can other things be defined as well. For example, `readOnly`, `writeOnly` etc.

There seems to be already solution present for this. Take a look at line. no. 203. For OpenAPI 3.1 we are already solving this usecase. If you can do the same for OpenAPI 3.0 (`schemaUtils31X.js` has code for 3.1 and `schemaUtils30X` should have for 3.0) we should be good. (outer properties should remain as is)

Reply:
@VShingala Merging the PR is not required now since OpenAPI 2.x, 3.0.x ignores sibling elements with ```$ref```. Closing the PR.